openapi/api-piattaforma-notifiche.md

```diff
@@ -0,0 +1,13 @@
+Procedura per importare le specifiche di PN:
```

Comment:
It's not clear for the naive reader when these operations should be run (ie. before a release?)

Do you think it would be useful (for devs) to provide an npm task (with some support script eventually) that implements all of these points?